meteor/server/api/playout/lib.ts

```diff
@@ -245,10 +245,20 @@ export async function setNextPart(
 		let newInstanceId: PartInstanceId
 		if (newNextPartInstance) {
 			newInstanceId = newNextPartInstance._id
+			cache.PartInstances.update(newInstanceId, {
+				$set: {
+					consumesNextSegmentId: newNextPart?.consumesNextSegmentId,
```

Comment:
we know in this route that `newNextPart` is always undefined, so this could be expressed as undefined/false instead